backend/cmd/parkserver/cmd/root.go

```diff
@@ -36,6 +38,13 @@ var rootCmd = &cobra.Command{
 			log.Warn().Msg("running in insecure mode")
 		}
 
+		// Establish a database connection
+		conn, err := pgx.Connect(context.Background(), dbURL)
```

Comment:
I mistakenly thought v5 did not support pools.  I've adjusted the code to re-introduce the pool.  Thanks for the catch.  Also added it to the config.  I was meaning to do these steps later in a separate PR for the DB integration, but I'll address them now since you mentioned them.